srcdeps-maven-local-repository/src/test/java/org/srcdeps/mvn/localrepo/SrcdepsLocalRepositoryManagerTest.java

```diff
@@ -138,6 +143,18 @@ public static void assertExists(Path path) {
     }
 
     @Test
+    public void mvnGitBom() throws Exception {
+        String project = "srcdeps-mvn-git-bom-quickstart";
+        assertBuild(project, "srcdeps-test-artifact", "0.0.2-SRC-revision-3d00c2a91af593c01c9439cb16cb5f52d2ddbcf8", new String[] {".pom"}, "clean", "install");
+
+        final String quickstartRepoDir = "org/srcdeps/mvn/quickstarts/" + project + "/" + project;
+        String dependentVersion = "1.0-SNAPSHOT";
+        final String artifactPrefix = quickstartRepoDir + "/" + dependentVersion + "/" + project + "-" + dependentVersion;
+        assertExists(mvnLocalRepo.resolve(artifactPrefix + ".jar"));
```

Comment:
> Why would there be a jar output for a BOM project?

The `artifactPrefix` var contains the path to the present quicstart's artifact. The present quickstart is called `srcdeps-mvn-git-bom-quickstart` and its packaging is `jar` https://github.com/srcdeps/srcdeps-maven/pull/2/commits/329de610163c35f8c7b57ad8085e2facbfb0ceb7#diff-8a294428db53c56f5c084c8a73dd1a98R27 

The existence of the BoM in the local repo is checked by `assertBuild` - see https://github.com/srcdeps/srcdeps-maven/pull/2/commits/329de610163c35f8c7b57ad8085e2facbfb0ceb7#diff-4998b579ab72ed1f342eac39e77b59afR148 where ask just for checking the `.pom` file.